docker/nginx/loris.nginx.conf

```diff
@@ -6,8 +6,8 @@ http {
   server {
     listen 9000;
 
-    location ~ ^/image/V[0-9]{4}.* {
-      rewrite ^/image/(V([0-9]{4}).*) /V$2000/$1 break;
+    location ~ ^/image/V[0-9]\{4\}.* {
+      rewrite ^/image/(?<filename>(?<shard>V[0-9]\{4\}).*) /s3:\k<shard>000/\k<filename> break;
```

Comment:
Could we get a comment explaining what the original URLs look like, and what we're trying to rewrite them too? That would make the regex much easier to follow.